### PR TITLE
Pixel Descent

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Pixel Descent

## Game Information
- **Title**: Pixel Descent
- **URL**: https://jc-bailey.itch.io/pixel-descent

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by Cebion — APPROVED** [review]

File: ports/pixeldescent/Pixel Descent.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Prepare files
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+[ -f "./gamedata/*.exe" ] && rm -f ./gamedata/*.exe
+
+# Display loading splash
+$ESUDO ./tools/splash "splash.png" 2000
+
+# Assign configs and load the game
+$GPTOKEYB "gmloadernext.aarch64" -c "pixeldescent.gptk" &
+pm_platform_helper "gmloadernext.aarch64"
```
Comment (kloptops) [inline ports/pixeldescent/Pixel Descent.sh:51]: This should be the full path.
```suggestion
pm_platform_helper "$GAMEDIR/gmloadernext.aarch64"
```

**Review by kloptops — CHANGES_REQUESTED** [review]

File: ports/pixeldescent/Pixel Descent.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Variables
+GAMEDIR="/$directory/ports/pixeldescent"
+
+# CD and set permissions
+cd $GAMEDIR
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+$ESUDO chmod +x -R $GAMEDIR/*
```
Comment (cdeletre) [inline ports/pixeldescent/Pixel Descent.sh:25]:
```suggestion
$ESUDO chmod +x $GAMEDIR/gmloadernext.aarch64
$ESUDO chmod +x $GAMEDIR/tools/splash
```

File: ports/pixeldescent/Pixel Descent.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  rm -Rf ./assets/
+  pm_message "Packing audiogroups complete"
+fi
+
+# Prepare files
+[ -f "./gamedata/data.win" ] && mv gamedata/data.win gamedata/game.droid
+[ -f "./gamedata/*.exe" ] && rm -f ./gamedata/*.exe
+
+# Display loading splash
+$ESUDO ./tools/splash "splash.png" 2000
```
Comment (cdeletre) [inline ports/pixeldescent/Pixel Descent.sh:47]:
```suggestion
$ESUDO ./tools/splash "splash.png" 2000 &
```

File: ports/pixeldescent/pixeldescent/lib/armeabi-v7a/libc++_shared.so
Comment (cdeletre) [inline ports/pixeldescent/pixeldescent/lib/armeabi-v7a/libc++_shared.so:1]: can be removed

File: ports/pixeldescent/pixeldescent/lib/armeabi-v7a/libcompiler_rt.so
Comment (cdeletre) [inline ports/pixeldescent/pixeldescent/lib/armeabi-v7a/libcompiler_rt.so:1]: can be removed

File: ports/pixeldescent/pixeldescent/lib/armeabi-v7a/libm.so
Comment (cdeletre) [inline ports/pixeldescent/pixeldescent/lib/armeabi-v7a/libm.so:1]: can be removed

File: ports/pixeldescent/pixeldescent/gmloadernext.aarch64
Comment (cdeletre) [inline ports/pixeldescent/pixeldescent/gmloadernext.aarch64:1]: Please add [license](https://github.com/PortsMaster/PortMaster-New/blob/bfc4fdd388344efdc65edced36c1002c93a7e334/ports/darkdeity/darkdeity/license/LICENSE.gmloader.md)

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**